harshita/level1/grade.cpp

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+    double marks,grade;
+    char A,B,C,D,E,F;
+    cout << "Enter your marks: ";
+    cin>> marks ;
+    if(marks > 80){
+        grade=A;
```

Comment:
first of all, A, B, C, ... they are variable not constant. if you want to do this you can do `grade='A'`, but you are directly printing so there is no need of this line as well.